app/assets/javascripts/application.js

```diff
@@ -1,3 +1,4 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
```

Comment:
Please remove this extra newline -- it's a little nit picky but we don't need this file to be changed in the commit history. It will make our lives easier later if we have to revert changes.